apps/meteor/ee/app/livechat-enterprise/server/api/monitors.ts

```diff
@@ -29,7 +32,10 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'livechat/monitors/:username',
-	{ authRequired: true },
+	{
+		authRequired: true,
+		permissionsRequired: { GET: { permissions: ['view-livechat-manager', 'manage-livechat-units'], operation: 'hasAny' } },
```

Comment:
Since we have a permission check here, so can we remove it from `findMonitorByUsername` to avoid re-checking?